handler/src/main/java/io/netty/handler/ssl/ReferenceCountedOpenSslEngine.java

```diff
@@ -1475,19 +1469,22 @@ public void run() {
         public void run(final Runnable runnable) {
             if (isDestroyed()) {
                 // The engine was destroyed in the meantime, just return.
-                runnable.run();
```

Comment:
@normanmaurer intentionally not running the runnable in this refactor?

Reply:
Yes as the SSL object is already destroyed